pkg/retester/retester.go

```diff
@@ -22,6 +22,14 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+type githubClient interface {
+	GetCombinedStatus(org, repo, ref string) (*github.CombinedStatus, error)
+	GetRef(string, string, string) (string, error)
+	QueryWithGitHubAppsSupport(ctx context.Context, q interface{}, vars map[string]interface{}, org string) error
+	CreateComment(owner, repo string, number int, comment string) error
+}
+
+// PullRequest represents GitHub PR and number of retests.
```

Comment:
PullRequest need not to be public, right?

Reply:
Correct, I will chakge it to private.